src/constants.rs

```diff
@@ -29,3 +29,15 @@ pub static KAKAROT_RPC_CONFIG: LazyLock<KakarotRpcConfig> =
 /// The RPC configuration.
 pub static RPC_CONFIG: LazyLock<RPCConfig> =
     LazyLock::new(|| RPCConfig::from_env().expect("failed to load RPC config"));
+
+/// Relayers addresses.
+pub static RELAYERS: LazyLock<Vec<Felt>> = LazyLock::new(|| {
+    let relayers = std::env::var("RELAYERS_ADDRESSES")
+        .expect("missing RELAYER_ADDRESSES in env")
```

Comment:
I just thought, I see that `RELAYERS_ADDRESSES` is missing from the `.env.example`, shouldn't we add it?